src/DSNConfigurator.php

```diff
@@ -108,4 +109,41 @@ private function applyConfig(PHPMailer $mailer, $config)
 
         return $mailer;
     }
+
+    /**
+     * Configure SMTP.
+     *
+     * @param PHPMailer $mailer PHPMailer instance
+     * @param array     $config Configuration
+     *
+     * @return PHPMailer
+     */
+    private function configureSMTP($mailer, $config)
+    {
+        $isSMTPS = 'smtps' === $config['scheme'];
```

Comment:
I note from their changelog:

> STARTTLS cannot be enabled anymore (it is used automatically if TLS is disabled and the server supports STARTTLS)
> [BC BREAK] Removed the encryption DSN option (use smtps instead)
> Added support for the smtps protocol (does the same as using smtp and port 465)

The first item here is also what PHPMailer does, though it can be disabled as it can cause breaks on localhost servers that advertise the STARTTLS extension. So I think your approach of defaulting to SMTPS on port 465 if they use `smtps` is fine.

Reply:
@Synchro, thanks for the help! Do I need to change/improve something else?